srtcore/core.cpp

```diff
@@ -10481,8 +10591,9 @@ SRT_REJECT_REASON CUDT::processConnectRequest(const sockaddr_any& addr, CPacket&
     }
     else
     {
-        SRT_REJECT_REASON error  = SRT_REJ_UNKNOWN;
-        int               result = s_UDTUnited.newConnection(m_SocketID, addr, packet, (hs), (error));
+        int error  = SRT_REJ_UNKNOWN;
+        string streaminfo_msg;
+        int result = s_UDTUnited.newConnection(m_SocketID, addr, packet, (hs), (error), (streaminfo_msg));
```

Comment:
Why do you get the contents of `STREAM_ID`. To put it into the response handshake?
How was it done before, if it was?

Reply:
It wasn't. That's a new thing here. Previously the failure response handshake contained no extensions. This is copying it back in order to pass it in case when the user has set some contents here.

Reply:
So this is something that we decided not to do at the moment.
There is no precedure for this defined yet.